rootconf/etc/ztd

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
```

Comment:
This single file has 3-space indentation, 4-space indentation, and 6-space indentation.